monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Spam_Mitigation_The_Challenge_of_Private_and_Permissionless.txt

```diff
@@ -0,0 +1,55 @@
+
+So, spam bloating attacks with a clear motive are far more dangerous than those without a clear motive, and this is a thing that's very important I want to keep in mind. When you look at the experience, the attack on Monero was driven by a need to create a, exploit a vulnerability. You need to get the block size to a certain size to do that. In Bitcoin we are seeing attacks now even though fees are very high, again, the reason is there's money to be made. So we have to look at the mode. Increasing the mixin and possibly increasing minimum relay fee, I think, are reasonable mitigation aspects, especially if we get efficiencies in ring signatures, as has been discussed.
+
+Okay, so questions and discussion.
+
+Audience member: You mentioned briefly the attack on anonymity, it's kind of been discussed in the last couple of months, of bloating ring sizes with outputs that you own. Do you think that if changing the fee structure to be more prohibitive to bloating block sizes with your own outputs is the way to solve this or should we be doing something else?
+
+Articmine: Again, this is an example of a targeted attack, and the example with a targeted attack is privacy. So what I would look at is, in that particular attack, would be increasing the ring size, because that immediately increases the cost to the attacker. If you, for example, have, let's say twice as much as what we have, twenty fake outputs instead of ten, well it's going to take twice as much spam to achieve the same level of deanonymization. And it's a relatively, comparatively cheap solution, if for example we look at a lot of the other research that's coming out in the pipeline with respect to efficiencies and ring signatures. So, to me that's the easy way to defend Monero right now, mainly because of, you also address other issues. It's also I think a compounding attack, I don't standalone it is as effective. But when combined with some of the other things that have been mentioned, then it's potentially more dangerous. So again if you address some of these other issues, such as the fee recommendations, to eliminate a fee of it's own essentially is what we want to get rid of. Then, again, we can tone that down that way.
+The one fee you can look at, somewhat, if you want to increase it, is the lowest fee. That is doable. I mean, you can even take it out entirely, and just set the minimum fee just slightly below the normal fee. That's a possibility, you're going to get about a five times gain just out of that on the maintenance. But I would say the real answer lies in addressing the motive, and the motive is attacking the privacy side of the coin, in that particular case.
+
```

Comment:
Please remove all the following newlines. Leaving just one would be fine

Reply:
Okay, I will add something else before pushing. Will do this in a moment.